xml/System.Device.Location/GeoCoordinateWatcher.xml

```diff
@@ -201,7 +201,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- Call <xref:System.Device.Location.GeoCoordinateWatcher.Dispose%2A> when you are finished using the <xref:System.Device.Location.GeoCoordinateWatcher> class?qualifyHint=False&autoUpgrade=True.  
+ Call <xref:System.Device.Location.GeoCoordinateWatcher.Dispose%2A> when you are finished using the <xref:System.Device.Location.GeoCoordinateWatcher> class.  
```

Comment:
For some reason this change isn't showing on stage but it looks good to me. @rpetrusha was having some issues with that too.
https://review.docs.microsoft.com/en-us/dotnet/api/system.device.location.geocoordinatewatcher.dispose?view=netframework-4.7.1&branch=pr-en-us-3757

Reply:
Yeah, this was driving me nuts.

Reply:
@mairaw - Thank you for the thoughtful comments and for the time spent reviewing! Much appreciated!!!